src/evidently/analyzers/data_profile_analyzer.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+# coding: utf-8
+from typing import Dict, List
+from typing import Optional
+from typing import Union
+
+from dataclasses import dataclass
+import numpy as np
+import pandas as pd
+
+from evidently import ColumnMapping
+from evidently.analyzers.base_analyzer import Analyzer
+from evidently.analyzers.utils import process_columns, DatasetUtilityColumns
+# from evidently.options import QualityMetricsOptions
+# from evidently.analyzers.utils import process_columns
+# from evidently.analyzers.utils import DatasetColumns
+
+
+@dataclass
+class DataProfileAnalyzerResults:
+    utility_columns: DatasetUtilityColumns
+    cat_feature_names: List[str]
+    num_feature_names: List[str]
+    datetime_feature_names: List[str]
+    ref_features_stats: Dict[str, Dict]
```

Comment:
in other results we use full names for `reference_` and `current_`, maybe here better to use full words too